### PR TITLE
ci(lint): automatically sort package.json files

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Instead of manually sorting stuff inside package.json files, I added something ([`sort-package-json`](https://www.npmjs.com/package/sort-package-json)) that will automatically sort package.json files on pre-commit when they are staged. Now all our package.json files will always stay ordered in the same way.

### Potential Breaking Changes

None

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,18 +3,47 @@
...
     "private": true,
     "scripts": {
+        "build": "ultra --recursive build",
+        "clean": "pnpm --recursive --parallel clean",
+        "commit-cli": "cz",
         "preinstall": "npx only-allow pnpm",
+        "lintfix": "pnpm --recursive --parallel lintfix",
+        "precommit": "lint-staged",
+        "prepare": "husky install",
+        "reconstruct": "npx rimraf node_modules packages/*/node_modules && npm run setup && echo done",
```
Comment (aiheon) [inline package.json:13]:
```suggestion
        "reconstruct": "npx rimraf node_modules packages/*/node_modules && pnpm run setup && echo done",
```
no? 👀 🤔

**Comment by gdostie** [comment]
The ordering of some fields (for example `license` and `author`) seem to be something else than alphabetical, but at least its always ordered the same. Dependencies seem to have the proper alphabetical order thought

File: package.json (hunk 1 above)
Comment (gdostie) [reply package.json:13]: Out of scope, I didn't touch that

**Review by aiheon — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (aiheon) [reply package.json:13]: I know I just found it weird 😆

**Review by GermainBergeron — APPROVED** [review]

**Pull request merged**